$NetBSD: patch-configure.ac,v 1.1 2016/09/15 09:25:53 maya Exp $

Add check for pthread_mutexattr_setprotocol.

Don't try to enable dtags in the wildcard case (i.e. non-linux),
netbsd passes the compile test, but fails at runtime (PR pkg/51324)

--- configure.ac.orig	2016-06-10 14:04:01.000000000 +0000
+++ configure.ac
@@ -891,9 +891,6 @@ case $host_os in
                            [LDRPATH_INSTALL="-Wl,-R,\\\$\$ORIGIN/\`\$(MAKEDEP) -R \${bindir} \${libdir}\`"
                             LDRPATH_LOCAL="-Wl,-R,\\\$\$ORIGIN/\$(top_builddir)/libs/wine"])])
 
-      WINE_TRY_CFLAGS([-Wl,--enable-new-dtags],
-                      [LDRPATH_INSTALL="$LDRPATH_INSTALL -Wl,--enable-new-dtags"])
-
       case $host_cpu in
         *i[[3456789]]86* | x86_64)
           WINE_TRY_CFLAGS([-Wl,-Ttext-segment=0x7bc00000],
@@ -1452,6 +1449,8 @@ then
         [AC_CHECK_HEADERS(pulse/pulseaudio.h,
             [AC_CHECK_LIB(pulse, pa_stream_is_corked,[:],[PULSE_LIBS=""],[$PULSE_LIBS])],
             [PULSE_LIBS=""])])
+	AC_CHECK_FUNCS(pthread_mutexattr_setprotocol, [HAVE_PTHREAD_MUTEXATTR_SETPROTOCOL=1],
+	[HAVE_PTHREAD_MUTEXATTR_SETPROTOCOL=0])
 fi
 WINE_NOTICE_WITH(pulse, [test -z "$PULSE_LIBS"],
         [libpulse ${notice_platform}development files not found or too old, Pulse won't be supported.],
